aggregate.go

```diff
@@ -78,19 +84,32 @@ type spanGroup struct {
 	Slowest []spanGroupSlowest // N-slowest traces in the group.
 }
 
+func (s spanGroup) Len() int      { return len(s.Slowest) }
+func (s spanGroup) Swap(i, j int) { s.Slowest[i], s.Slowest[j] = s.Slowest[j], s.Slowest[i] }
+func (s spanGroup) Less(i, j int) bool {
+	a := s.Slowest[i]
+	b := s.Slowest[j]
+	return a.End.Sub(a.Start) > b.End.Sub(b.Start)
```

Comment:
Would it be less clear to use `<` and then wrap it in a [`sort.Reverse`](https://godoc.org/sort#Reverse) adapter? I am guessing it would be less clear, but just want to confirm.

> so it can be read as "A sorts before B if it took a greater amount of time than B".

That'd make a good comment.